extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -14,23 +14,94 @@
  * limitations under the License.
  */
 
+.i-amphtml-story-interactive-slider-container {
+  --percentage: 0%;
+}
+
 .i-amphtml-story-interactive-slider-input-container {
+  display: flex !important;
   background: var(--i-amphtml-interactive-option-background-color) !important;
   border-radius: 2em !important;
   box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
+  align-items: center !important;
+  justify-content: center !important;
+  height: 100% !important;
+  width: 100% !important;
```

Comment:
You can set `height: 3em` and don't set the width, and it will work

Reply:
I don't think height or width are needed